### PR TITLE
[TASK] Move PHP linting to GitHub actions

### PR DESCRIPTION
**Pull request by andreaswolf**
Resolves: #12

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          - 7.2
+          - 7.3
+          - 7.4
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Run PHP lint
+      run: composer run-script ci:php:lint
```
Comment (oliverklee) [inline .github/workflows/ci.yml]: Why not the shorter form?

`composer ci:php:lint`

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Composer script for PHP linting
 
 ### Changed
+- Moved PHP linting to GitHub actions (#12)
```
Comment (oliverklee) [inline CHANGELOG.md]: … and also #28

Comment (andreaswolf) [reply CHANGELOG.md]: I don't see why we need PR + issue ID, esp. since the PR ID can't be known beforehand.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (andreaswolf) [reply .github/workflows/ci.yml]: true… I think that's a leftover from the GitHub action template I had a look at.

**Pull request merged**